docs/sdk-and-tools/notifier.md

```diff
@@ -0,0 +1,248 @@
+---
+id: notifier
+title: Notifier
+---
+
+## Overview
+
+A MultiversX observer node can push block events to a notifier service, which will process
```

Comment:
please add the notifier page into `docs/sdk-and-tools/overview.md` as well

Reply:
added